src/Core/Grid/Column/ColumnCollection.php

```diff
@@ -107,14 +107,39 @@ public function toArray()
         return $columns;
     }
 
+    /**
+     * Swap two existing columns.
+     *
+     * @param string $keyFrom the Column ID original position in the Collection
+     * @param string $keyTo the Column ID destination position in the Collection
+     *
+     * @return self
+     */
+    public function swap($keyFrom, $keyTo)
+    {
+        if (!array_key_exists($keyFrom, $this->items) || !array_key_exists($keyTo, $this->items)) {
+            throw new ColumnNotFoundException(sprintf(
+                'Cannot swap two existing columns from collection. Columns with ids "%s" or "%s" were not found.',
```

Comment:
The error message is confusing, you can't say "two existing columns" when at least one is missing

Reply:
oups, to answer that: this is what happens when we build exception messages in classes not built for :/ 

Honest: I think it's good enough for a developer to figure out which id has been malformed

Reply:
I'd replace `Cannot swap two existing columns from collection` with `Cannot swap nonexistent columns from collection` which seems clearer

Reply:
I've updated: wdyt?

Reply:
Solved as we have changed from swap() to move()